.github/workflows/pull-request-token.yml

```diff
@@ -82,6 +79,43 @@ jobs:
           path: "target/deploy/*.so"
           if-no-files-found: error
 
+  cargo-test-bpf-twoxtx:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set env vars
+        run: |
+          echo "RUST_STABLE_VERSION=1.60.0" >> $GITHUB_ENV
```

Comment:
How is this used?

Reply:
Ahhh, thanks. I should have just looked at that file.
I wonder if we can find a way to pull this from `solana` instead of having to hard code it. Let's go with this for now, though, to unblock CI and set up the separate job